tutorials/plot_visualize_raw.py

```diff
@@ -49,9 +56,13 @@
 # We can check where the channels reside with ``plot_sensors``. Notice that
 # this method (along with many other MNE plotting functions) is callable using
 # any MNE data container where the channel information is available.
-raw.plot_sensors(kind='3d', ch_type='mag')
+raw.plot_sensors(kind='3d', ch_type='mag', regions='position')
 
 ###############################################################################
+# We used ``regions='position'`` to color code the different regions. This is
+# useful for checking if the ``order='position'`` of
+# :func:`raw.plot <mne.io.Raw.plot>` separated the regions correctly.
```

Comment:
It's probably worth doing `raw.plot(order='selection')` below to show it to people. Plus MNE-C people might like seeing it.